app/src/main/java/org/simple/clinic/summary/PatientSummaryConfig.kt

```diff
@@ -1,6 +1,25 @@
 package org.simple.clinic.summary
 
+import io.reactivex.Observable
+import org.simple.clinic.remoteconfig.ConfigReader
+import org.threeten.bp.Duration
+
 data class PatientSummaryConfig(
     val numberOfBpPlaceholders: Int,
-    val numberOfBpsToDisplay: Int
-)
+    val numberOfBpsToDisplay: Int,
+    val bpEditableDuration: Duration
+) {
+
+  companion object {
+
+    fun read(configReader: ConfigReader): Observable<PatientSummaryConfig> {
+      val bpEditableDurationConfig = configReader.long("bp_editable_duration_in_minutes", 60)
```

Comment:
This will read from the disk, so maybe we should use `Observable.fromCallable {}` instead and inline the reading from the `ConfigReader` in there?